src/test/java/.gitignore

```diff
@@ -0,0 +1,17 @@
+# Package Files #
+.classpath
+.project
+.settings/
+target/
+
+# IntelliJ IDEA files
+.idea/
+*.iws
+*.iml
+*.ipr
+
+### Gradle
+.gradle/
+out/
+bin/
+build/
```

Comment:
If there are some new exclusions here, maybe we could add them to the .gitignore we have at the root of the project and delete this second gitignore file?

Reply:
That's a good idea thanks i will compare this with the one we have and add the missing bits 👍